src/content/en/2021/performance.md

```diff
@@ -1,20 +1,534 @@
+  )
+}}
+
+LCP was faster on desktop than mobile. TTFB affects LCP like FCP.  Comparisons by device, connection type, and rank all mirror the trends of FCP. Render-blocking resources, total weight, and loading strategies all affect LCP performance.
+
+{{ figure_markup(
+  image="performance-LCP-by-ect.png",
+  caption="LCP performance by connection type",
+  description="Stacked bar chart showing LCP performance for effective connection type. For offline websites, 49% had good, 17% needs improvement, and 34% poor performance. For slow 2G, 1%  were needs improvement and 99% poor. For 2G, 4% were needs improvement and 96% poor. For 3G, 8% were good, 28% needs improvement, and 65% poor. For 4G, 51% were good, 33% needs improvement, and 16% poor.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=583947675&format=interactive",
+  sheets_gid="730149797",
+  sql_file="web_vitals_by_eff_connection_type.sql"
+  )
+}}
+
+Offline origins with good LCP more closely matched 4G experiences, though poor LCP experiences were higher for offline. LCP occurs after FCP, and the additional budget of 0.7 seconds could be why more offline websites achieved good LCP than FCP.
+
+{{ figure_markup(
+  image="performance-LCP-by-rank.png",
+  caption="LCP performance by rank",
+  description="Stacked bar chart showing LCP performance for each ranking group. For the top 1,000 websites, 64% had good, 25% needs improvement, and 11% poor performance. For the top 10,000, 59% were good, 30% needs improvement, and 11% poor. For the top 100,000, 55% were good, 32% needs improvement, and 13% poor. For the top 1,000,000, 53% were good, 33% needs improvement, and 14% poor. For all, 50% were good, 33% needs improvement, and 17% poor.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=1080361066&format=interactive",
+  sheets_gid="730149797",
+  sql_file="web_vitals_by_rank.sql"
+  )
+}}
+
+For LCP, the differences in performance by rank were closer than FCP. Also, a higher proportion of origins in the top 1,000 had poor LCP. On WordPress, <a hreflang="en" href="https://datastudio.google.com/s/kvq1oJ60jaQ">28% of origins experienced good LCP</a>. This is an opportunity to improve user experience as poor LCP is usually caused by a handful of problems.
+
+### The LCP Element
+
+Let's take a deeper dive into the LCP element.
+
+{{ figure_markup(
+  image="performance-top-15-lcp-nodes.png",
+  caption="Top 15 LCP HTML element nodes",
+  description="Bar chart showing the percent share of pages for each node type for the LCP element- data listed by desktop then mobile: IMG 47% 42%, DIV 28% 27%, P 6% 10%, H1 3% 5%, SECTION 3% 2%, undetected 2% 2%, H2 2% 2%, A 1% 2%, SPAN 1% 1%, HEADER 1% 1%, LI 1% 1%, RS-SBG 1% 1%, H3 0% 1%, TD 1% 1%, VIDEO 0% 0%",
```

Comment:
I don't think the code formatting is needed since it's not actually code - just a listing of the node names. It's probably a web component.

Reply:
Part of the reason for code formatting is the readability as otherwise the numbers in the headings bleed into the percentages:

![image](https://user-images.githubusercontent.com/10931297/141794622-ae102a4a-2394-4ca8-8b90-a1d6d8203ad7.png)

Here's what it looks like with the code formatting:

![image](https://user-images.githubusercontent.com/10931297/141794843-9bef87f3-743a-4778-9e0d-2f3001b40ff5.png)

(still don't like the lighter code formatting on the dark grey background - but that's a completely different issue and not your problem!)